src/main/java/org/opensearch/neuralsearch/processor/chunker/DelimiterChunker.java

```diff
@@ -69,4 +56,15 @@ public List<String> chunk(String content) {
 
         return chunkResult;
     }
+
+    /**
+     * Return the chunked passages for delimiter algorithm
+     *
+     * @param content input string
+     * @param parameters a map containing parameters, containing the following parameters
+     */
+    @Override
+    public List<String> chunk(String content, Map<String, Object> parameters) {
```

Comment:
No need to override this method. We call the method without runtime parameters in delimiter algorithm.

Reply:
Done.